python/feathub/processors/spark/dataframe_builder/spark_dataframe_builder_constants.py

```diff
@@ -0,0 +1,15 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+TIME_ATTRIBUTE_NAME = "__time_attribute__"
```

Comment:
Since there is no reason for this constant string value to be different from `EVENT_TIME_ATTRIBUTE_NAME` in `flink_table_builder_constants.py`, would it be simpler to move `feathub/processors/constants.py`?